core/src/geometry/point.rs

```diff
@@ -157,6 +157,24 @@ impl Point {
         Point::new(self.x.abs(), self.y.abs())
     }
 
+    /// Offsets a point by subtracting a size.
+    ///
+    /// This method provides a workaround for the `Sub` trait not being usable in `const` contexts.
+    ///
+    /// # Panics
+    ///
+    /// This function will panic if `width` or `height` are too large to be represented as an `i32`
+    /// and debug assertions are enabled.
+    pub const fn sub_size(self, other: Size) -> Point {
```

Comment:
IMO we shouldn't make the workaround methods public. If a method is only used in `embedded-graphics` and not in `embedded-graphics-core` it could be moved to the `PointExt` trait.

Reply:
I don't think that'll work because trait functions can't be `const`, so I changed them to be `pub(crate)`.

Reply:
OK, if the method is only used in one crate that's the better solution anyway.